server/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    Controller,
+    Get,
+    Post,
+    Body,
+    UseGuards,
+    Request
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { LoginDto } from './auth.dto';
+import { JwtAuthGuard } from 'src/auth/jwt-auth.guard';
+
+//TODO: return roles associated as well when that relation has been established
+const SELECT = {
+    id: true,
```

Comment:
Please format this file with prettier

This is something we should work on but we should add prettier formatting automatiicaly as a precommit hook or a github action, we want to avoid tickets with code formatting.